Satellite-im/UplinkWeb
# chore(delete): add delete account button on auth page

### What this PR does 📖

- Adds delete button on unlock page on the bottom right side when user is on dev mode


https://github.com/user-attachments/assets/97cb9e68-15fa-4f16-b3e0-d0918f39f723



How to test
- sign up
- no delete button should appear while signing up if dev mode is not activated
- go to profile
- activated dev mode
- go to profile
- logout
- should appear delete button
- login 
- remove dev mode
- logout
- login 
- button should not appear
